### ISSUE TITLE
Firefox only plays 8 or 16 bit wav files

### ISSUE THREAD

**Issue by Evert-R** [Amsterdam-Music-Lab/MUSCLE#130]
Firefox doesn't play 24 or 32 bit wave files. No error is raised, there's just no sound.

**Comment by BeritJanssen** [comment]
Perhaps we should put this somewhere in the documentation?

**Comment by jaburgoyne** [comment]
This is related to the chat I started in our internal Teams: could we write some hard specifications for recommended browser and OS configurations? Or at least a list of configurations that we know will *not* work?

**Comment by drikusroor** [comment]
As we now working on adding validation to playlists, we might also consider allowing the user to only upload 8 & 16 bit wav files.

**Comment by jaburgoyne** [comment]
Yes, for sure. In general, I think it is best to prefer compressed formats for faster streaming, but WAV is always going to be necessary for some more particular experiments where lossy compression is untrustworthy
